apps/docs/src/app/platform/component-docs/platform-link/platform-link-examples/platform-link-disabled-example.component.html

```diff
@@ -0,0 +1,8 @@
+<p>
+    This is an Example of
+    <fdp-link [href]="'http://www.google.com'" [title]="'Extra info as tooltip text and aria-label'" [disabled]="true">
+        <fd-icon [glyph]="'chain-link'" [size]="'l'"></fd-icon>DisabledLink</fdp-link
+    >
```

Comment:
not sure if the formatting is right.. </fdp-link is in one line and > on another, is this expected?

Reply:
This formatting is coming from Visual code IDE on saving the file. It may be expected as lint as passing it.

Reply:
have done proper formatting now